targets/posix/nanoCLR/CLRStartup.cpp

```
@@ -0,0 +1,544 @@
+        memset(&g_CLR_RT_EventCache, 0, sizeof(g_CLR_RT_EventCache));
+        memset(&g_CLR_RT_GarbageCollector, 0, sizeof(g_CLR_RT_GarbageCollector));
+        memset(&g_CLR_HW_Hardware, 0, sizeof(g_CLR_HW_Hardware));
+
+        for (auto &it : m_assemblies)
+            delete it.second;
+        m_assemblies.clear();
+
+        m_fInitialized = false;
+    }
+
+    Settings() : m_configureRuntimeCallback(nullptr), m_fInitialized(false), m_configured(false)
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: m_clrOptions, m_assemblies, m_configureRuntimeCallback

(cppcoreguidelines-pro-type-member-init)